r4.0/comps-dom0.xml

```diff
@@ -50,16 +50,17 @@
       <packagereq>firewalld</packagereq>
```

Comment:
You updated r4.0 comps file for dom0 with content of r4.1 one. This is wrong.

Reply:
Ok. That was not very clear at the beginning if it was good synced comps or not. The same for the below one. I'll update in consequence.